static/app/components/organizations/timeRangeSelector/utils.tsx

```diff
@@ -66,5 +66,8 @@ export function getRelativeSummary(
     }
   }
 
-  return defaultRelativePeriodString;
+  // @TODO I think this is wrong. relativeOptions argument. If
+  // relativeOptions[relative] resolves to some non-string value, calling .toLowerCase
+  // in places like components/stream/group.tsx should throw an error.
+  return defaultRelativePeriodString as string;
```

Comment:
@matejminar any ideas here (I see you added the default here)? I dont know how this passed #e697816 the tsc compiler then, I get errors now

Reply:
Sorry for the late reply, just got back to work after 3 weeks. If I recall correctly, it was done because of this: https://github.com/getsentry/sentry/blob/ae19ca7edc6170a81008b7bfd7ce981651427bb4/static/app/views/releases/detail/overview/index.tsx#L464-L493

Therefore the value can definitely be something truthy and non-string. That is not correct. Looks like the `defaultRelativePeriodString` const was of type `any`, that's why it passed the compiler before.

Reply:
No worries :) Hope you enjoyed the time off! I'll reopen a PR with these fixes :)